modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
 
+### Changed
+- Separated the sinks and backstage setups between dialogs and other popups. #TINY-9223
```

Comment:
This doesn't seem like a user-facing change? We probably don't need a changelog until we have the final scroll container fix 🤔

Reply:
Should the changelog have only user-facing changes? Isn't that the difference between the changelog and the release notes? The changelog might impact someone contributing, whereas the release notes is for the user?

Reply:
It's always been our rule for anything customer facing it should only contain things that impact them. 🤷

Reply:
Fair enough. I don't mind either way. Just trying to understand the boundaries and reasons.

Reply:
I should add at least one of the reasons are to avoid confusing users with things they won't understand. As remember this is effectively what community use to know what changed in TinyMCE when upgrading.

Reply:
- [ ] remove non user-facing entry from changelog